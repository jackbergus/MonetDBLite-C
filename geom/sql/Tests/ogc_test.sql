-- Conformance Item N1
SELECT f_table_name
FROM geometry_columns;
-- Conformance Item N2
SELECT g_table_name
FROM geometry_columns;
-- Conformance Item N3
SELECT storage_type
FROM geometry_columns
WHERE f_table_name = 'streams';
-- Conformance Item N4
SELECT geometry_type
FROM geometry_columns
WHERE f_table_name = 'streams';
-- Conformance Item N5
SELECT coord_dimension
FROM geometry_columns
WHERE f_table_name = 'streams';
-- Conformance Item N6
SELECT max_ppr
FROM geometry_columns
WHERE f_table_name = 'streams';
-- Conformance Item N7
SELECT srid
FROM geometry_columns
WHERE f_table_name = 'streams';
-- Conformance Item N8
SELECT srtext
FROM SPATIAL_REF_SYS
WHERE SRID = 101;
