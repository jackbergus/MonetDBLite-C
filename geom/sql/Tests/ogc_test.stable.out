stdout of test 'ogc_test` in directory 'geom/sql` itself:


# 15:41:01 >  
# 15:41:01 >  "mserver5" "--debug=10" "--set" "gdk_nr_threads=0" "--set" "mapi_open=true" "--set" "mapi_port=31341" "--set" "mapi_usock=/var/tmp/mtest-4281/.s.monetdb.31341" "--set" "monet_prompt=" "--forcemito" "--set" "mal_listing=2" "--dbpath=/export/scratch1/mk/current//Linux/var/MonetDB/mTests_geom_sql" "--set" "mal_listing=0"
# 15:41:01 >  

# MonetDB 5 server v11.18.0
# This is an unreleased version
# Serving database 'mTests_geom_sql', using 8 threads
# Compiled for x86_64-unknown-linux-gnu/64bit with 64bit OIDs dynamically linked
# Found 15.591 GiB available main-memory.
# Copyright (c) 1993-July 2008 CWI.
# Copyright (c) August 2008-2014 MonetDB B.V., all rights reserved
# Visit http://www.monetdb.org/ for further information
# Listening for connection requests on mapi:monetdb://vienna.ins.cwi.nl:31341/
# Listening for UNIX domain connection requests on mapi:monetdb:///var/tmp/mtest-4281/.s.monetdb.31341
# MonetDB/GIS module loaded
# MonetDB/JAQL module loaded
# MonetDB/SQL module loaded

Ready.

# 15:41:01 >  
# 15:41:01 >  "mclient" "-lsql" "-ftest" "-Eutf-8" "-i" "-e" "--host=/var/tmp/mtest-4281" "--port=31341"
# 15:41:01 >  

#SELECT f_table_name
#FROM geometry_columns;
% sys.geometry_columns # table_name
% f_table_name # name
% varchar # type
% 15 # length
[ "shore_gid"	]
[ "centerline_gid"	]
[ "centerlines_gid"	]
[ "boundary_gid"	]
[ "position_gid"	]
[ "centerline_gid"	]
[ "position_gid"	]
[ "footprint_gid"	]
[ "shores_gid"	]
[ "boundary_gid"	]
[ "neatline_gid"	]
#SELECT storage_type
#FROM geometry_columns
#WHERE f_table_name = 'streams';
% sys.geometry_columns # table_name
% storage_type # name
% int # type
% 1 # length
#SELECT geometry_type
#FROM geometry_columns
#WHERE f_table_name = 'streams';
% sys.geometry_columns # table_name
% geometry_type # name
% int # type
% 1 # length
#SELECT coord_dimension
#FROM geometry_columns
#WHERE f_table_name = 'streams';
% sys.geometry_columns # table_name
% coord_dimension # name
% int # type
% 1 # length
#SELECT max_ppr
#FROM geometry_columns
#WHERE f_table_name = 'streams';
% sys.geometry_columns # table_name
% max_ppr # name
% int # type
% 1 # length
#SELECT srid
#FROM geometry_columns
#WHERE f_table_name = 'streams';
% sys.geometry_columns # table_name
% srid # name
% int # type
% 1 # length
#SELECT srtext
#FROM SPATIAL_REF_SYS
#WHERE SRID = 101;
% sys.spatial_ref_sys # table_name
% srtext # name
% varchar # type
% 386 # length
[ "PROJCS[\"UTM_ZONE_14N\", GEOGCS[\"World Geodetic System\n72\",DATUM[\"WGS_72\", ELLIPSOID[\"NWL_10D\", 6378135,\n298.26]],PRIMEM[\"Greenwich\",\n0],UNIT[\"Meter\",1.0]],PROJECTION[\"Transverse_Mercator\"],\nPARAMETER[\"False_Easting\", 500000.0],PARAMETER[\"False_Northing\",\n0.0],PARAMETER[\"Central_Meridian\", -99.0],PARAMETER[\"Scale_Factor\",\n0.9996],PARAMETER[\"Latitude_of_origin\", 0.0],UNIT[\"Meter\", 1.0]]"	]

# 15:41:01 >  
# 15:41:01 >  "Done."
# 15:41:01 >  

